cmake/addLibrary.cmake

```diff
@@ -94,8 +94,8 @@ MACRO(ALPAKA_ADD_LIBRARY libraryName)
             ENDFOREACH()
             ADD_LIBRARY(
                 ${libraryName}
-                ${sourceFileNames}
                 ${libraryType}
+                ${sourceFileNames}
```

Comment:
PLease move source files as last argument to fulfill the Cmake description https://cmake.org/cmake/help/latest/command/add_library.html